File: Angular/src/app/models/productincart.ts
export class  productincart{
    productId:number;
    productName:string;
    categoryId:number;
    companyId:Number;
    description:string;
    matchAge:number;
    price:number;
    picture:String;
    lastUpdatedDate:Date;
    quantity:string;
    totalPrice:string;

    constructor( productId:number, productName:string, categoryId:number, companyId:Number,description:string,matchAge:number,price:number,  picture:String, lastUpdatedDate:Date,quantity:string,totalPrice:string)
    {
        this.productId=productId;
        this.productName=productName;
        this.companyId=companyId;
        this.categoryId=categoryId;
        this.description=description;
        this.matchAge=matchAge;
        this.price=price;
        this.picture=picture;
        this.lastUpdatedDate=lastUpdatedDate;
        this.quantity=quantity;
        this.totalPrice=totalPrice;

    }
}

File: C#/פרויקט סופי!/Dal/Models/ProductInCart.cs
using System;
using System.Collections.Generic;

namespace Dal_Repository.Models;

public partial class ProductInCart
{
    public int ProductId { get; set; }

    public string ProductName { get; set; } = null!;

    public int CategoryId { get; set; }

    public int CompanyId { get; set; }

    public string? Description { get; set; }

    public int? MatchAge { get; set; }

    public decimal? Price { get; set; }

    public string? Picture { get; set; }

    public DateTime LastUpdatedDate { get; set; }

    public int Quantity { get; set; }

    public decimal? TotalPrice { get; set; }

    public virtual Category Category { get; set; } = null!;

    public virtual Company Company { get; set; } = null!;
}
